### PR TITLE
S&L Definitions Release

### PR DESCRIPTION
**Pull request by ss-evoco**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: benefit/data/sheets/WorkBasketResultFields/WorkBasketResultFields-sandl-nonprod.json [hunk 1]
```diff
@@ -1,15 +1,9 @@
...
-  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "workBasketHearingDate", "Label": "Hearing Date", "DisplayOrder": 18},
-  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-judge", "CaseFieldID": "workBasketHearingDate", "Label": "Hearing Date", "DisplayOrder": 18},
-  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-dwpresponsewriter", "CaseFieldID": "workBasketHearingEpimsId", "Label": "Hearing Venue", "DisplayOrder": 19},
-  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "workBasketHearingEpimsId", "Label": "Hearing Venue", "DisplayOrder": 19},
-  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "workBasketHearingEpimsId", "Label": "Hearing Venue", "DisplayOrder": 19},
-  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-judge", "CaseFieldID": "workBasketHearingEpimsId", "Label": "Hearing Venue", "DisplayOrder": 19}
+
```
Comment (DGordonV1) [inline benefit/data/sheets/WorkBasketResultFields/WorkBasketResultFields-sandl-nonprod.json:2]: Empty newlines here

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab.json [hunk 2]
```diff
@@ -233,6 +233,7 @@
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "elements_and_issues", "TabLabel": "Elements and issues", "TabDisplayOrder": 23, "CaseFieldID": "elementsDisputedChildElement", "TabFieldDisplayOrder": 7},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "elements_and_issues", "TabLabel": "Elements and issues", "TabDisplayOrder": 23, "CaseFieldID": "elementsDisputedChildDisabled", "TabFieldDisplayOrder": 8},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "elements_and_issues", "TabLabel": "Elements and issues", "TabDisplayOrder": 23, "CaseFieldID": "elementsDisputedLimitedWork", "TabFieldDisplayOrder": 9},
+
```
Comment (DGordonV1) [inline benefit/data/sheets/CaseTypeTab/CaseTypeTab.json:236]: Empty new line, does this break formatting / sheets at all?

**Review by DGordonV1 — CHANGES_REQUESTED** [review]

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab.json (hunk 2 above)
Comment (ss-evoco) [reply benefit/data/sheets/CaseTypeTab/CaseTypeTab.json:236]: Its done to break out the sections. It wont break it, lots of the file has them.

**Review by DGordonV1 — APPROVED** [review]

**Review by damien-cooke — APPROVED** [review]

File: benefit/data/sheets/CaseEventToFields/CaseEventToFields.json [hunk 3]
```diff
@@ -168,6 +213,17 @@
     "PageFieldDisplayOrder": 3,
     "PageID": "1.0"
   },
+  {
+    "LiveFrom": "06/10/2022",
+    "CaseEventID": "updateListingRequirements",
+    "CaseFieldID": "reserveTo",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "OPTIONAL",
```
Comment (amoslv1) [inline benefit/data/sheets/CaseEventToFields/CaseEventToFields.json:221]: This has been changed to complex for https://tools.hmcts.net/jira/browse/SSCS-10822
```suggestion
    "DisplayContext": "COMPLEX",
```

File: benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json [hunk 4]
```diff
@@ -1276,5 +1276,52 @@
...
+  {"LiveFrom": "05/09/2020", "CaseTypeID": "Benefit", "CaseFieldID": "postponementEvent", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
+  {"LiveFrom": "05/09/2020", "CaseTypeID": "Benefit", "CaseFieldID": "postponementEvent", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
+  {"LiveFrom": "23/11/2022", "CaseTypeID": "Benefit", "CaseFieldID": "adjournmentInProgress", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
+  {"LiveFrom": "23/11/2022", "CaseTypeID": "Benefit", "CaseFieldID": "adjournmentInProgress", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
+  {"LiveFrom": "23/11/2022", "CaseTypeID": "Benefit", "CaseFieldID": "adjournmentInProgress", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
+  {"LiveFrom": "23/11/2022", "CaseTypeID": "Benefit", "CaseFieldID": "adjournmentInProgress", "UserRole": "caseworker-sscs-judge", "CRUD": "CRUD"},
+  {"LiveFrom": "10/03/2022", "CaseTypeID": "Benefit", "CaseFieldID": "reserveTo", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
+  {"LiveFrom": "10/03/2022", "CaseTypeID": "Benefit", "CaseFieldID": "reserveTo", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"}
```
[the remark this answers is not in the document]
Comment (amoslv1) [reply benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json:1326]: can wait for 10822 for this

File: benefit/data/sheets/CaseEventToFields/CaseEventToFields.json (hunk 3 above)
Comment (amoslv1) [reply benefit/data/sheets/CaseEventToFields/CaseEventToFields.json:221]: Can wait for 10822

File: benefit/data/sheets/EventToComplexTypes/EventToComplexTypes.json [hunk 5]
```diff
@@ -186,6 +186,26 @@
...
+    "ListElementCode": "duration",
+    "ID": "updateListingRequirementsOverrideFieldsDuration",
+    "DisplayContext": "OPTIONAL",
+    "LiveFrom": "01/07/2022",
+    "FieldDisplayOrder": 1,
+    "PageID": "1.0"
+  },
+  {
+    "CaseEventID": "updateListingRequirements",
+    "CaseFieldID": "reservedTo",
```
Comment (amoslv1) [inline benefit/data/sheets/EventToComplexTypes/EventToComplexTypes.json:201]: This should be "reserveTo" but it is being changed in 10822 so could wait for that?

File: benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json [hunk 6]
```diff
@@ -17,6 +17,9 @@
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "reasonableAdjustmentsOutstanding", "Label": "Alternative letter format required", "DisplayOrder": 16},
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "hasUnprocessedAudioVideoEvidence", "Label": "Does case have unprocessed audio/video evidence?", "DisplayOrder": 17},
   {"LiveFrom": "16/12/2021", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "dwpDueDate", "Label": "FTA response due date", "DisplayOrder": 18},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting Officer attending?", "DisplayOrder": 19},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "workBasketHearingDate", "Label": "Hearing Date", "DisplayOrder": 19},
+  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "workBasketHearingEpimsId", "Label": "Hearing Venue", "DisplayOrder": 20},
```
Comment (amoslv1) [inline benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json:22]: DisplayOrder seems wrong here and throughout the rest of the additions to this file

File: benefit/data/sheets/WorkBasketResultFields/WorkBasketResultFields.json [hunk 7]
```diff
@@ -9,19 +9,28 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "interlocReviewState", "Label": "Interlocutory review state", "DisplayOrder": 8},
   {"LiveFrom": "25/09/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "linkedCasesBoolean", "Label": "Has related appeal(s)?", "DisplayOrder": 9},
   {"LiveFrom": "25/09/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "[LAST_STATE_MODIFIED_DATE]", "Label": "Date of state change", "DisplayOrder": 10, "ResultsOrdering": "1:ASC"},
+  {"LiveFrom": "25/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting Officer attending?", "DisplayOrder": 11},
+  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "workBasketHearingDate", "Label": "Hearing Date", "DisplayOrder": 18},
+  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "workBasketHearingEpimsId", "Label": "Hearing Venue", "DisplayOrder": 19},
```
Comment (amoslv1) [inline benefit/data/sheets/WorkBasketResultFields/WorkBasketResultFields.json:14]: Same again, display order doesn't look right

**Review by amoslv1 — DISMISSED** [review]
Double check display orders
Question about reserveTo

File: benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json (hunk 6 above)
Comment (ss-evoco) [reply benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json:22]: Will leave as this was tested this way and passed SIT.

File: benefit/data/sheets/WorkBasketResultFields/WorkBasketResultFields.json (hunk 7 above)
Comment (ss-evoco) [reply benefit/data/sheets/WorkBasketResultFields/WorkBasketResultFields.json:14]: Will leave as this was tested this way and passed SIT.

**Review by amoslv1 — APPROVED** [review]

**Pull request merged**